test/unit/SpatialReferenceTest.cpp

```diff
@@ -391,3 +391,16 @@ TEST(SpatialReferenceTest, test_bounds)
     EXPECT_FLOAT_EQ(static_cast<float>(b2.maxy), 39.01261687f);
 
 }
+
+TEST(SpatialReferenceTest, issue_1989)
+{
+    SpatialReference srs;
```

Comment:
Maybe add a test case for `EPSG:32732`, which is a common style of UTM zone that many use to ensure it always passes. Also, please add a comment in the test in reference to what issue 1989 is. If our ticket system were to migrate or move away, knowing a tiny bit of context so that people can get by without having to dereference to the ticket system is helpful.